pages/common/jq.md

```diff
@@ -0,0 +1,23 @@
+# jq
+
+> A lightweight and flexible command-line JSON processor.
+
+- Output JSON file:
+
+`jq '' {{file}}`
```

Comment:
I end up doing `cat <file> | jq` most of the time also. However, I wanted to call out that you need to add the empty query if you invoke jq directly, since this is a common error that users run into initially.

The empty string is also technically required if you pipe an empty jq query into another command (e.g. `cat <file> | jq | grep <text>` will not work, but `cat <file> | jq '' | grep <text>` will). Admittedly this usage is not so common either...